File: get_gids.py
import re

with open(r'C:\Users\Ducbv\.gemini\antigravity\brain\bd67975a-acb4-4a7b-947f-d1c370182b00\.system_generated\steps\178\content.md', 'r', encoding='utf-8') as f:
    content = f.read()

# Pattern for Google Sheets JSON data containing sheet IDs and names
# Look for structures like: [1,0,"1823144076",[{"1":[[0,0,"Aging"]
pattern = r'\[\d+,\d+,\"(\d+)\",\[\{\"1\":\[\[0,0,\"([^\"]+)\"'
matches = re.findall(pattern, content)

for gid, name in matches:
    print(f"{name}: {gid}")

File: get_gids.js
const fs = require('fs');
const content = fs.readFileSync('C:/Users/Ducbv/.gemini/antigravity/brain/bd67975a-acb4-4a7b-947f-d1c370182b00/.system_generated/steps/178/content.md', 'utf8');
const regex = /\[\d+,\d+,"(\d+)",\[\{"1":\[\[0,0,"([^"]+)"/g;
let match;
while ((match = regex.exec(content)) !== null) {
  console.log(match[2] + ': ' + match[1]);
}
